### PR TITLE
CLDR-14320 clarify ordering of keys in extensions

### PR DESCRIPTION
**Pull request by macchiati**
Also fixes typo in the BNF for 'keyword'

##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14320
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Review by FrankYFTang — APPROVED** [review]
LGTM

File: docs/ldml/tr35.md [hunk 1]
```diff
@@ -3532,8 +3532,8 @@ To canonicalize the syntax of _source_:
   * Put any variants into alphabetical order (eg, en-fonipa-scouse, not en-scouse-fonipa)
   * Put any extensions into alphabetical order by their singleton (eg, en-t-xxx-u-yyy, not en-u-yyy-t-xxx)
   * Put all attributes into alphabetical order.
-  * Put all <keywords, tfields> pairs into alphabetical order of their keys, within their respective extensions.
-  * Remove any type or tfield value of "true"
+  * Put all ufields (<ukey, uvalue>) and tfields (<tkey, tvalue>) into alphabetical order according to their keys (ukey or tkey), within their respective extensions.
+  * Remove any uvalue (aka type) equal to "true"
```
Comment (FrankYFTang) [inline docs/ldml/tr35.md:3536]: 1. Please add "." to the end of the sentence.
2. Could we explicit mention the "true" of tvalue cannot be removed due to the constrains of tfield definition here?

Comment (FrankYFTang) [inline docs/ldml/tr35.md:3536]: 1. Please add "." to the end of the sentence.
2. Could we explicit mention the "true" of tvalue cannot be removed due to the constrains of tfield definition here?

**Review by FrankYFTang — CHANGES_REQUESTED** [review]

File: docs/ldml/tr35.md (hunk 1 above)
Comment (macchiati) [reply docs/ldml/tr35.md]: done

**Review by FrankYFTang — APPROVED** [review]
LGTM

**Pull request merged**